README.md

```diff
@@ -6,4 +6,23 @@ This is a collection of links that are helpful reading for people joining the Gu
 
 * [GitHub flow](https://guides.github.com/introduction/flow/:w)
 * [Creating use Git commit messages](http://chris.beams.io/posts/git-commit/)
-* [Git pickaxe](http://www.philandstuff.com/2014/02/09/git-pickaxe.html)
\ No newline at end of file
+* [Git pickaxe](http://www.philandstuff.com/2014/02/09/git-pickaxe.html)
+
+## Internal Guardian Pages
+* [Spike](http://spike/) - Noticeboard, canteen food, HR policies, lots more
+* [GEE/Taleo](https://gnm.taleo.net/) - objectives
+* [Oracle Self Service](http://oracle.dmz.gnl:8079/OA_HTML/AppsLogin) - payslips and holiday booking
+* [Big brother](http://octhelp/inventory/bigbrother) - for finding people ;) - only really works with people on desktops
+
+## Useful Docs
+* [Department admin guide](https://docs.google.com/document/d/1ErsZUEL0ELiGUYXHbEWlm0mLGY7lZoAnzF4IXWqnPG0/editr)
+* [Socials/Events/Activities](https://docs.google.com/document/d/1aHFEtlXG0f-R12S0SPBzLXa1JnRpIJYjactHaYjmyGk/edit)
+
+## Development tools
+*[TeamCity](https://teamcity.gutools.co.uk/)
```

Comment:
Maybe worth adding a note about what each of these do (like you've done for Janus)